src/core.ts

```diff
@@ -202,6 +202,16 @@ interface SiteAdapterOptions {
       (Enables special handling of certain HTML elements, which doesn't
        work well on all sites) */
   iframe?: boolean;
+
+  /**
+   * Styling function for the selected row
+   */
+  styleSelectedRow?(row: DataRow): void;
+
+  /**
+   * Styling function for the non-selected rows
+   */
+  styleUnselectedRows?(rows: DataRow[]): void;
```

Comment:
@TylerMillis thoughts on making this `styleUnselectedRow` and having it act on a single row, so it's more consistent with `styleSelectedRow`?

Reply:
I made this function more in line with the selected row function. It makes sense for them to be more uniform and easier to use.